packages/cli/test/util/load-plan.test.ts

```diff
@@ -271,3 +271,35 @@ test.serial('old-workflow: load a plan from workflow path', async (t) => {
     expression: 'x()',
   });
 });
+
+test.serial('step: allow file paths for state and data', async (t) => {
```

Comment:
For the record this test is testing two things at once: loading `state` as a file path and loading `state.data` as a file path.

That really ought to be two different tests.

Good unit tests are as localised and focused as possible